components/contribution-flow/graphql/queries.js

```diff
@@ -36,6 +43,10 @@ export const contributionFlowAccountQuery = gql`
             }
           }
         }
+        totalContributedToHost(inCollectiveCurrency: true) {
+          valueInCents
+          currency
+        }
```

Comment:
An alternative API if we want to support the tier currency thing mentioned above. We may need field aliases for that to work.

```suggestion
        totalContributedToHost(usingCurrencyFromTier: { legacyId: $tierId }) @include(if: $includeTier) {
          valueInCents
          currency
        }
        totalContributedToHost(inCollectiveCurrency: true) @skip(if: $includeTier) {
          valueInCents
          currency
        }
```